packages/image_picker/image_picker/test/image_picker_test.mocks.dart

```diff
@@ -22,8 +22,8 @@ import 'package:mockito/mockito.dart' as _i1;
 
 class _FakeLostData_0 extends _i1.Fake implements _i2.LostData {}
 
-class _FakeLostDataResponse_1 extends _i1.Fake
-    implements _i2.LostDataResponse {}
+class _FakeLostDataResponse_1 extends _i1.Fake implements _i2.LostDataResponse {
+}
```

Comment:
Please, revert this file, this pr has nothing to do with the image_picker package.